ts/test/test-worker-threads.ts

```diff
@@ -8,10 +8,10 @@ const assert = require('assert');
 
 describe('Worker Threads', () => {
   // eslint-ignore-next-line prefer-array-callback
-  it('should work when propagated to a worker through -r flag', function () {
+  it('should work when propagated to workers through -r flag', function () {
```

Comment:
This has nothing to do with your PR, but the comment for the test does not seem to match what it's actually tested (I don't see any `-r` flag).

Reply:
Me neither, but it works regardless. Just running the test from the command line, e.g.
```
node out/test/worker.js
```
runs as expected, `worker.ts` loads the profiler module on line 3 with:
```javascript
import {time} from '../src/index';
```